libs/pxweb2-ui/src/lib/components/Link/Link.stories.tsx

```diff
@@ -1,18 +1,123 @@
-import type { Meta } from '@storybook/react';
+import type { Meta, StoryObj, StoryFn } from '@storybook/react';
 import { Link } from './Link';
```

Comment:
Could we add a blank line between imports from packages and imports from within the project, to make it easier to quickly see where something is imported from?
```suggestion

import { Link } from './Link';
```